src/tracer.js

```diff
@@ -43,21 +64,15 @@ export const startTrace = async () => {
         path,
       });
 
+      if (isTimeoutTimerEnabled()) setupTimeoutTimer();
+
       const functionSpan = getFunctionSpan();
       logger.debug('startTrace span created', functionSpan);
 
-      if (!isSendOnlyIfErrors()) {
-        const { rtt } = await sendSingleSpan(functionSpan);
-        return addRttToFunctionSpan(functionSpan, rtt);
-      } else {
-        logger.debug(
-          "Skip sending start span because tracer in 'send only if error' mode ."
-        );
-        return null;
-      }
-    } else {
-      return null;
```

Comment:
so no sendOnlyIfErrors?

Reply:
We have protected in `sendSpans` I added also a test to `startTrace` with this case